renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -709,10 +709,47 @@ bool WrappedOpenGL::Serialise_glDrawArrays(GLenum mode, GLint first, GLsizei cou
   return true;
 }
 
+void WrappedOpenGL::CopyClientMemoryArrays(GLint first, GLsizei count)
+{
```

Comment:
Ah I already started on the rewrite before I read this. I actually think its better now - I have the early out for idle frames and when VAO != 0. Let me know if thats OK.

I know you didnt like the lambdas before, but given Ive now expanded their use, is it now justifiable? It allows keeping everything neatly in 1 function, including the VertexAttribData struct.